src/chess/board.cc

```diff
@@ -705,17 +705,11 @@ bool ChessBoard::HasMatingMaterial() const {
     return true;
   }
 
-  // All the pieces together.
```

Comment:
This change is unnecessary.

Reply:
Fixed, although this comment wasn't dismissed because of the vagaries of the fix/where the comment was planted